cassandra_migrate/migration.py

```diff
@@ -47,8 +47,11 @@ def load(cls, path):
             content = fp.read()
 
         checksum = bytes(hashlib.sha256(content.encode('utf-8')).digest())
-        return cls(os.path.abspath(path), os.path.basename(path), content,
-                   checksum)
+        # Should use enum but python3 requires importing an extra library
```

Comment:
I'm fine with including `enum34` for the Python 2 compat.

Reply:
I remember now why filetype. Initially I thought of using an enum to identify the script's extension. However this is truly a binary decision. Either cql or py. So, using a boolean sounds like a good solution. Unless you're planning on adding more script types (cql, py, ?) I think a boolean will work just fine. 

Thoughts?

Reply:
A boolean is fine, as long as it is named as such (maybe `is_python` or something similar as I suggested earlier).